### PR TITLE
feat: add mdiSvg option

### PR DESCRIPTION
**Pull request by morphatic**
Add `mdiSvg` to the list of options that can be selected when using the CLI to add Vuetify to a
project.

CAVEAT: I didn't know how to test this, and I'm not familiar at all with the API for Vue CLI plugins. But this is the option I go with more often than not when bootstrapping a project, so it seemed like it should be there.

Happy to fix, if necessary.

### CODE REVIEW

File: packages/vue-cli-plugin-vuetify/generator/tools/fonts.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ const fonts = {
     import: '@mdi/font/css/materialdesignicons.css',
     link: '<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">',
   },
+  mdiSvg: {
+    package: {
+      '@mdi/js': '^4.7.95',
```
Comment (jacekkarczmarczyk) [inline packages/vue-cli-plugin-vuetify/generator/tools/fonts.js]: Could be `'*'`, they are not following semantic versioning

Comment (morphatic) [reply packages/vue-cli-plugin-vuetify/generator/tools/fonts.js]: Good call. I've updated the PR to reflect this change.

File: packages/vue-cli-plugin-vuetify/generator/tools/fonts.js [hunk 2]
```diff
@@ -2,11 +2,16 @@ const helpers = require('./helpers')
 const fonts = {
   mdi: {
     package: {
-      '@mdi/font': '^3.6.95',
+      '@mdi/font': '*',
```
Comment (johnleider) [inline packages/vue-cli-plugin-vuetify/generator/tools/fonts.js:5]: we should use an explicit version here

File: packages/vue-cli-plugin-vuetify/generator/tools/fonts.js [hunk 3]
```diff
@@ -2,11 +2,16 @@ const helpers = require('./helpers')
...
     },
     import: '@mdi/font/css/materialdesignicons.css',
     link: '<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">',
   },
+  mdiSvg: {
+    package: {
+      '@mdi/js': '*',
```
Comment (johnleider) [inline packages/vue-cli-plugin-vuetify/generator/tools/fonts.js:12]: we should use an explicit version here

**Review by johnleider — CHANGES_REQUESTED** [review]

**Comment by johnleider** [comment]
This Pull Request is being closed due to inactivity.

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Pull request closed without merging**